ret-core/src/main/kotlin/io/rabobank/ret/IntelliSearch.kt

```diff
@@ -23,6 +23,13 @@ data class FilterRound(val filter: String, val startIndex: Int, val endIndex: In
     fun hasFailed() = endIndex <= 0
 }
 
+/**
+ * Match results like IntelliJ (to be injected)
+ *
+ * This class is used to match a string of characters to a possible candidate, based on partial string matching.
+ * Example: "as" will positively match with "Admin Service" or "admin-service", and so will "adser".
```

Comment:
Keep in mind that this displays as "one line": `Example: ...` will be appended to the previous line and so would `The matching ...`

Reply:
This is actually a generic comment, because I see this format in multiple locations and I don't think that's the actual format you're looking for.

Reply:
Good point. In most cases I added the extra lines for better reading in editors, but I'll double check if it is like I wanted it to be